src/basins/edgetracking.jl

```diff
@@ -0,0 +1,194 @@
+export edgetracking, bisect_to_edge
+
+"""
+    edgetracking(ds::DynamicalSystem, u1, u2, attractors::Dict; kwargs...)
+Runs the edge tracking algorithm[^1] for a multistable DynamicalSystem `ds`.
+
+The algorithm is initialized from two states `u1`, `u2` in state space that must belong to
+different basins of attraction. The system's `attractors` are specified as a dictionary of
+`StateSpaceSet`s.
+
+## Keyword arguments
+* `eps1 = 1e-7`: bisection distance threshold
+* `eps2 = 1e-6`: trajectory divergence distance threshold
+* `maxiter = 100`: maximum number of iterations before the algorithm stops
+* `abstol = 0.0`: convergence threshold for returned edge state (distance in state space)
+* `tmax = Inf`: maximum integration time of parallel trajectories until re-bisection 
+* `dt = 0.01`: integration time step
+* `ϵ_mapper = 0.1`: `ϵ` parameter in [`AttractorsViaProximity`](@ref)
+* `verbose = false`: if true, prints info while running
+* `output_level = 2`: controls what to return (see below)
+* `kwargs...`: additional keyword arguments to be passed to `AttractorsViaProximity`
+
+## Description
```

Comment:
I would like some more clarity on how the "tracking" works. I understand how the boundary is located via successive bijections. But this gives you an arbitrary point on the boundary. Why would the next step, that is called "tracking", lead to the edge state? Doesn't it imply some assumptions about the flow, specifically that the flow locally around the basin boundary is oriented towards the edge state? Is this assumption always true?

Reply:
Yes, this is a core assumption underlying the edge tracking algorithm. If the flow on the basin boundary converges to some attractor, then this attractor will be the edge state you find. Typically this is a saddle, in which case the basin boundary (or a subset of it) is its stable set. There can be multiple edge states along the basin boundary, and then the algorithm will find one of them, depending on where the initial bisection locates the boundary. It is also possible that the entire basin boundary forms the edge state. For example, consider an unstable limit cycle separating two basins. The algorithm will find some point on this limit cycle and then track along it in circles until it stops.

Reply:
Then you need to state this description in the `## Description` section of the docstring!